itemloaders/utils.py

```diff
@@ -19,14 +15,9 @@ def arg_to_iter(arg):
     """
     if arg is None:
         return []
-    elif (
-        hasattr(arg, '__iter__')
-        and not isinstance(arg, _ITERABLE_SINGLE_VALUES)
-        and not is_item(arg)
```

Comment:
It seems the docstring is no longer correct; it was also not really correct before the change as well. Would it make sense to fix it before merging?